File: app/src/main/java/com/sstechcanada/todo/activities/SettingsActivity.java
package com.sstechcanada.todo.activities;

import com.sstechcanada.todo.R;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        setTitle(getString(R.string.action_settings));
    }


}

File: app/src/main/java/com/sstechcanada/todo/activities/SettingsActivity.kt
package com.sstechcanada.todo.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.sstechcanada.todo.R

class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        title = getString(R.string.action_settings)
    }
}